xc/device.ml

```diff
@@ -1832,6 +1832,9 @@ let vnconly_cmdline ~info ?(extras=[]) domid =
     @ (List.fold_left (fun l (k, v) -> ("-" ^ k) :: (match v with None -> l | Some v -> v :: l)) [] extras)
 
 let vgpu_args_of_info info domid =
+	let path = sprintf "/var/lib/xen/demu-%s.%d" in
+	let suspend_file = path "save" domid in
+	let resume_file = path "resume" domid in
```

Comment:
This code could re-use `Device_common.demu_save_path` and `Device_common.demu_restore_path`.